ODP-Runtime.md

```diff
@@ -42,22 +43,22 @@ To help achieve TONE, ODP-compliant Hadoop platforms MUST conform to the followi
 Hadoop Version Specifications
 -----------------------------
 
--   ODP Platforms MUST be a descendent of the Apache Hadoop 2.7 branch.
+-   For this version of the specification, ODP Platforms MUST be a descendent of the Apache Hadoop 2.7 branch.  Future versions MAY increase the base Apache Hadoop version.
+
+-   The Apache components in an ODP reference release MUST have their source be 100% committed to an Apache source tree.
```

Comment:
Committed?  Do we want to allow for a patch to be available but not yet committed? 
Also, does this belong here?  Should the ODP specification be putting specifications on the ODP process itself?  If we do, we should at least discuss what ODP sees the relationship between the ODP reference release and the vendor releases being.